### PR TITLE
Update Calico to 3.19 (3.17.x branch)

### PR DESCRIPTION
**Pull request by tfussell**
Towards https://github.com/giantswarm/giantswarm/issues/17954

### CODE REVIEW

File: service/controller/cloudconfig/cloud_config.go [hunk 1]
```diff
@@ -24,39 +24,6 @@ const (
 iscsid.safe_logout = Yes`
 )
 
-const calicoKubeKillScript = `#!/bin/bash
```
[the remark this answers is not in the document]
Comment (yulianedyalkova) [reply service/controller/cloudconfig/cloud_config.go:27]: Going through the old issue, it looks like it got resolved in `v2.6.3` ([comment](https://github.com/projectcalico/calico/issues/1420#issuecomment-352044958)) and I think the problem not happening on a clean cluster so cncf won't catch it. So I'm not 100% removing this one is safe.

[the remark this answers is not in the document]
Comment (tfussell) [reply service/controller/cloudconfig/cloud_config.go:27]: Here's some info about how we can test it https://github.com/giantswarm/kvm-operator/pull/775#issuecomment-623303462

[the remark this answers is not in the document]
Comment (tfussell) [reply service/controller/cloudconfig/cloud_config.go:27]: To test, I'm creating a cluster at release 14.1.2, upgrading it to 15.0.0, deleting the master, and checking connectivity via manual tests, prometheus metrics (net-exporter), and a CNCF test.

[the remark this answers is not in the document]
Comment (tfussell) [reply service/controller/cloudconfig/cloud_config.go:27]: No errors reported by net-exporter metrics. Pod to pod, pod to node, and pod to service communication looks to be working after restarting the master. I'll run conformance tests overnight.

[the remark this answers is not in the document]
Comment (tfussell) [reply service/controller/cloudconfig/cloud_config.go:27]: Conformance reported no issues as well. Going with this.

**Pull request merged**